lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/AzureBatchBootstrapREEFLauncher.java

```diff
@@ -95,20 +88,26 @@ public static void main(final String[] args) throws IOException, InjectionExcept
         TANG.newInjector(generateConfiguration(jobSubmissionParameters))
             .getInstance(AzureBatchBootstrapDriverConfigGenerator.class);
 
+    LocalAddressProvider defaultLocalAddressProvider =
+        Tang.Factory.getTang().newInjector().getInstance(LocalAddressProvider.class);
```

Comment:
Don't get the concrete class. Use the ILocalAddressProvider instead.

Reply:
In java, LocalAddressProvider is the interface name. They don't have the 'I' notation for interfaces.

---
In reply to: [199966934](https://github.com/sharathmalladi/reef/pull/6#discussion_r199966934) [](ancestors = 199966934)